src/components/Snippet/FileManager.vue

```diff
@@ -148,62 +151,59 @@ export default {
 </script>
 
 <template lang="pug">
-.girder-data-browser-snippet
-  v-row
-    v-col
-      v-card
```

Comment:
@matthewma7 what's the reason for the `v-row`, `v-col`, `v-card` nesting here?  I removed them and everything looks the same.

Reply:
Hm, I don't remember, but please go ahead and remove it if it works fine without them.